### DISCUSSION TITLE
How to complete a database from a user's CSV file?

### DISCUSSION THREAD

**Discussion by Oh-Nue** [Budibase/budibase#4397] [Ideas]
Hello, 

I would like to know if it is possible via the "Attachment" field for a user to import a CSV file and to fill a database with the content from this file. The columns in the CSV file would match the columns in the database. 

If this is possible, could you tell me how to do it :)

Thank you in advance!

**Comment by mjashanks** [comment]
Sorry, this is not possible at the moment.

In future, I think this should be done via automations. I.e. send file to an automation, which reads the rows and inserts/updates.

Converting this to an "Idea" as it is a good one!

**Reply by Oh-Nue** [reply]
Thank you for this answer! Should I close the topic or leave it like that? :)

**Reply by mjashanks** [reply]
Just leave it! It's useful for us to look back on.
